vizro-core/examples/visual-vocabulary/pages/examples/gantt.py

```diff
@@ -0,0 +1,30 @@
+import pandas as pd
+import plotly.graph_objects as go
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+from vizro.models.types import capture
+
+tasks = pd.DataFrame(
+    [
+        {"Task": "Job A", "Start": "2009-01-01", "Finish": "2009-02-28"},
+        {"Task": "Job B", "Start": "2009-03-05", "Finish": "2009-04-15"},
+        {"Task": "Job C", "Start": "2009-02-20", "Finish": "2009-05-30"},
+    ]
+)
+
+
+@capture("graph")
+def gantt(data_frame: pd.DataFrame, x_start: str, x_end: str, y: str) -> go.Figure:
+    fig = px.timeline(data_frame, x_start, x_end, y)
+    fig.update_yaxes(autorange="reversed")
+    return fig
+
+
+page = vm.Page(
+    title="Gantt",
+    components=[vm.Graph(px.gantt(tasks, x_start="Start", x_end="Finish", y="Task"))],
+)
```

Comment:
For simplicity, I would remove the creation of a custom chart and then just insert the chart directly here. Let's just change the order in the dataframe instead 👍 

```suggestion
page = vm.Page(
    title="Gantt",
    components=[vm.Graph(px.timeline(tasks, x_start="Start", x_end="Finish", y="Task"))],
)
```